### DISCUSSION TITLE
other implementations

### DISCUSSION THREAD

**Discussion by hwuebben** [ultralytics/yolov5#6846] [Q&A]
does anybody know a yolo v5 implementation with a better code quality?

**Comment by glenn-jocher** [comment]
@hwuebben hi, thank you for your feature suggestion on how to improve YOLOv5 🚀!

The fastest and easiest way to incorporate your ideas into the official codebase is to submit a Pull Request (PR) implementing your idea, and if applicable providing before and after profiling/inference/training results to help us understand the improvement your feature provides. This allows us to directly see the changes in the code and to understand how they affect workflows and performance.

Please see our ✅ [Contributing Guide](https://github.com/ultralytics/yolov5/blob/master/CONTRIBUTING.md) to get started.
